packages/spectral-config/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@readme/spectral-config",
+  "version": "1.0.0",
+  "description": "ReadMe coding standards for API documentation",
+  "main": "index.js",
+  "files": [
+    "index.js"
+  ],
+  "scripts": {
+    "lint": "eslint .",
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/readmeio/standards.git"
+  },
+  "author": "Engineers <engineers@readme.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/readmeio/standards/issues"
+  },
+  "homepage": "https://github.com/readmeio/standards",
```

Comment:
Yeah might be a good idea

Reply:
I'm going to leave this alone because the page on NPM will look like garb with a long URL.
![Screen Shot 2022-11-04 at 10 30 23 AM](https://user-images.githubusercontent.com/33762/200039015-e492e4a7-6b19-4397-8f3a-12ae0499b599.png)